llvm/test/CodeGen/AMDGPU/amdgcn.bitcast.ll

```diff
@@ -1946,8 +1946,21 @@ end:
   ret void
 }
 
-
-
+; CHECK-LABEL: {{^}}v_bitcast_v32f16_to_v8i64:
+define <8 x i64> @v_bitcast_v32f16_to_v8i64(<32 x half> %a, i32 %b) {
```

Comment:
Also add the inverse case if it's not already there, and the with `s/half/bfloat/` and